index.html

```diff
@@ -49,9 +50,21 @@
             </li>
           </ul>
         </div>
+        <div class="mobile__nav">
+          <div class="mobile__logo"><img src="./img/mobile-logo.svg" alt="mobile logo" /></div>
+          <div class="responsive__navbar">
+            <div class="burgermenu">
+              <img src="/img/white-line.svg" alt="line" />
```

Comment:
iti sugerez sa faci un link cu clasa de hamburger (a) in care sa ai un div mare, iar liniile le faci din span-uri pe care setezi width si height, culoare etc.
```suggestion
              <span class="hamburger__line></span>
```